serverservice/methods.go

```diff
@@ -0,0 +1,132 @@
+package serverservice
```

Comment:
Future-proof this as the `fleetdb` package?

Reply:
yep, I'll take this on separately